Modules/_collectionsmodule.c

```diff
@@ -2194,7 +2194,7 @@ defdict_init(PyObject *self, PyObject *args, PyObject *kwds)
 }
 
 PyDoc_STRVAR(defdict_doc,
-"defaultdict(default_factory[, ...]) --> dict with default factory\n\
+"defaultdict(default_factory=None, /, [...]) --> dict with default factory\n\
 \n\
 The default factory is called without arguments to produce\n\
```

Comment:
is called if it's not None

Reply:
I think that this is implied.  In any case, this issue is not about rewriting the doc string.